src/Symfony/Component/Console/Helper/Table.php

```diff
@@ -237,13 +244,27 @@ public function setHeaders(array $headers)
         return $this;
     }
 
+    /**
+     * Sets the rows.
+     *
+     * @param array $rows
+     *
+     * @return Table
+     */
     public function setRows(array $rows)
     {
         $this->rows = array();
 
         return $this->addRows($rows);
     }
 
+    /**
+     * Adds a list of rows, each row must must be an array or a TableSeparator instance.
```

Comment:
The description of the param should be removed as it is/should be described in `@param`. The description should be removed altogether IMO as it does not add anything we cannot deduce from the method name.